compiler/include/llvmUtil.h

```diff
@@ -82,5 +82,8 @@ bool isOpaquePointer(llvm::Type* ty);
 // from it. Otherwise, return nullptr.
 llvm::Type* tryComputingPointerElementType(llvm::Value* ptr);
 
+llvm::Type* getPointerType(llvm::LLVMContext& ctx, unsigned AS=0);
```

Comment:
Let's add a comment to these two that says it returns the equivalent of a `void*` in C; but that newer LLVMs don't distinguish between pointer types.